[CodeSec RN] 23.5.2

anaghapamidi opened the pull request.

anaghapamidi committed:
[CodeSec RN] 23.5.2

anaghapamidi committed:
Add image

anaghapamidi commented:
Summary of Changes:

Modified files:
- cspm/rn/book.yml
- cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-2023.adoc
- cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc

New Files:

- cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-may-2023.adoc
- codesec-rn-23.5.2.png

Tickets: 
- RLP-102020
- RLP-102779

Pubs-MV commented on cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc:
```diff
@@ -22,15 +22,23 @@ NOTE: The details and functionality listed below are a preview and the actual re
+|===
+2+|Deprecation Notice
+
+|*Projects legacy page*
```
Can we call this Project 1.0? instead of legacy?

any suggestions @tsmithv11

tsmithv11 replied:
We never called it Projects 1.0 in public pages/documentation, so I don't think users will understand. "Legacy" or "Non-enhanced" would be more clear.

Pubs-MV commented on cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc:
```diff
@@ -22,15 +22,23 @@ NOTE: The details and functionality listed below are a preview and the actual re
+
+|*Projects legacy page*
+//RLP-102779 - From 23.5.2 no longer supported.
+|From 23.5.2 Prisma Cloud will no longer support Projects legacy page. If you have not already been using the enhanced Projects page, from this release you will automatically be directed here to continue monitoring your code errors and vulnerabilities.
```
Starting in 23.5.2, the Enhanced Projects page replaces the 1.0 version where you can continue monitoring your code errors and vulnerabilities.

anaghapamidi committed:
Update NF Dashboard with Gilad's feedback

tsmithv11 commented on cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-may-2023.adoc:
```diff
@@ -0,0 +1,30 @@
+
+Learn about the new Code Security capabilities on Prisma™ Cloud Enterprise Edition (SaaS) in May 2023.
+
+The following new features or enhancements are available for Prisma Cloud Code Security. These capabilities help agile teams add security checks to their existing IaC (Infrastructure-as-Code) model and enforce security throughout the build lifecycle.
```
```suggestion
The following new features or enhancements are available for Prisma Cloud Code Security. These capabilities help agile teams add security checks to their development process and enforce security throughout the build lifecycle.
```

Not just IaC anymore :)

tsmithv11 commented on cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-may-2023.adoc:
```diff
@@ -0,0 +1,30 @@
+|DESCRIPTION
+
+|*Code Security Dashboard*
+|Code Security dashboard (*Dashboard > Code Security*) enables you to analyze error trends across code integrations on Prisma Cloud. While emphasizing on the critical and high severity code errors, the dashboard also  gives you an understanding of the potentially vulnerable vectors in repositories, policy errors including secrets, non-compliant package licenses, CVE vulnerabilities and IaC miscon ...
```
```suggestion
|Code Security dashboard (*Dashboard > Code Security*) enables you to analyze error trends across code integrations on Prisma Cloud. While emphasizing the critical and high severity code issues, the dashboard also gives you an understanding of the potentially vulnerable vectors in repositories, policy errors including secrets, non-compliant package licenses, vulnerabilities and IaC misconfigurations. See the historical trend for Code Issues and Pull Requests to make informed decisions to prevent the recurrence of the trend.
```

tsmithv11 commented on cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-may-2023.adoc:
```diff
@@ -0,0 +1,30 @@
+
+|*Code Security Dashboard*
+|Code Security dashboard (*Dashboard > Code Security*) enables you to analyze error trends across code integrations on Prisma Cloud. While emphasizing on the critical and high severity code errors, the dashboard also  gives you an understanding of the potentially vulnerable vectors in repositories, policy errors including secrets, non-compliant package licenses, CVE vulnerabilities and IaC miscon ...
+The vectors for repositories, policy errors, non-compliant package licenses, CVE vulnerabilities and IaC misconfigurations ensure you see the latest real time data after scans.
```
"CVE vulnerabilities" is redundant. Is that common for us to use or should it be either/or?

Pubs-MV replied:
CVE is vulnerabilities

Pubs-MV committed:
Update look-ahead-planned-updates-prisma-cloud-code-security.adoc

moved the change in existing behavior to the May RN

Pubs-MV committed:
Update cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-may-2023.adoc

Co-authored-by: Taylor <28880387+tsmithv11@users.noreply.github.com>

Pubs-MV committed:
Update cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-may-2023.adoc

Co-authored-by: Taylor <28880387+tsmithv11@users.noreply.github.com>

Pubs-MV committed:
Update features-introduced-in-code-security-may-2023.adoc

added the change in default behavior here instead of the look ahead because were not really providing a deprecation notice when the change for projects is going live.

Pubs-MV reviewed (commented):
@anaghapamidi I made some edits without talking to you because we had no time for review.
